source/preferences/troubleshoot-notifications.rst

```diff
@@ -0,0 +1,144 @@
+Troubleshoot notifications
+==========================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+The Mattermost notifications you receive depend on your `Mattermost preferences <#check-your-mattermost-preferences>`__, the `Mattermost client <#check-your-mattermost-client-settings>`__ you're using, and the `operating system (OS) <#check-your-operating-system-settings>`__ you're running Mattermost on.
+
+If you’re having trouble receiving Mattermost notifications, see the following sections for troubleshooting steps you can follow to ensure you're receiving the notifications you want.
+
+Check your Mattermost preferences
+----------------------------------
+
+Start by ensuring that your Mattermost preferences have notifications enabled:
+
+1. Select **Settings** |gear| in the top right corner of Mattermost.
+2. Select **Desktop and mobile notifications**.
+
+If **Send notifications for** is set to **Nothing**, then Mattermost notifications are currently disabled. Select either the **All new messages** or **Mentions, direct messages, and group messages** option instead.
+
+We recommend checking your Mattermost client settings next.
+
+Check your Mattermost client settings
+-------------------------------------
+
+.. tab:: Desktop
+
+    Ensure notifications are enabled in your Mattermost server connection.
+
+    1. Select the Mattermost Server option in the top left of the desktop app, then edit the server details.
+    2. Under **Permissions**, enable **Notifications**.
+
+    [add an animated GIF here]
+
+.. tab:: Mobile
+
+    Ensure that your Mattermost mobile preferences have notifications enabled.
```

Comment:
Don't these belong in the section above `Check your Mattermost preferences`? I think these are not "client settings" but Mattermost preferences when the _client_ is _Mobile_.

Reply:
@abhijit-singh - I asked myself the very same question and decided to simplify the instructions and story we're trying to tell with this structure. Very open to further discussion with the UX on this if you feel that this approach will introduce confusion.

Reply:
I do think it might be confusing to see the instructions in the first section when you visit this page from the mobile app. Here's what the first scroll would look like in the current structure:
<img width="401" alt="image" src="https://github.com/user-attachments/assets/998448f4-5375-43db-b079-ac0c8ec1b87e">

I think we should distinguish between instructions for `Desktop` vs `Mobile` with tabs in this section since instructions to check Mattermost Settings for Mobile are significantly different.